cloudant-sync-datastore-core/src/test/java/com/cloudant/sync/internal/documentstore/DatabaseImplRevsDiffTest.java

```diff
@@ -26,7 +26,7 @@
 public class DatabaseImplRevsDiffTest extends BasicDatastoreTestBase{
 
     @Test
-    public void revsDiff_emptyInput_returnEmpty() {
+    public void revsDiff_emptyInput_returnEmpty() throws Exception {
```

Comment:
It could be `DocumentStoreException` but with it only being test code I didn't think it mattered too much.